### ISSUE TITLE
News Release detail page: adopt V1 components where components are not used

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#17068]
## Description
To be in compliance with the Design System and to help make sure our products receive design and accessibility updates, News Release templates needs to be updated to use the most recent versions of all available components.

Example pages:

- [Minneapolis News Release List](https://www.va.gov/minneapolis-health-care/news-releases/)
  - [News Release Detail page](https://www.va.gov/minneapolis-health-care/news-releases/minneapolis-va-health-care-system-media-advisory/)
- [Boston News Release List](https://www.va.gov/boston-health-care/news-releases/)
  - [News Release Detail page](https://www.va.gov/boston-health-care/news-releases/va-boston-participates-in-national-age-friendly-health-systems-initiative/)

### Elements that are not using components that should be (Backlog)

**News Release List template & News Release Detail template**
1. Link
   - News release titles are currently not using a component at all but they should be using [the v1 component](https://design.va.gov/components/link) 
   - "See all news releases" on the detail template should use the [active link component](https://design.va.gov/components/link/#active)
2. ~Back to top
I think the News Release Detail pages might meet the requirements for the [Back to top component](https://design.va.gov/components/back-to-top), unless there is a specific reason that we aren't using it here?~

~Pagination~
   - ~**NOTE: This is for News Release List templates**~   
   - ~These are currently not using a component at all but they should be using [the v3 component](https://design.va.gov/components/pagination#examples---v3)~
  

## Acceptance Criteria
- [ ] Link component has been added 
- [ ] Update and run existing test(s).
- [ ] If additional tests are needed, stub a ticket to note this for future work.
- [ ] Requires accessibility review

**Comment by maxx1128** [comment]
I estimate this at a **2.** News Releases are a relatively simple template, so there shouldn't be any issues.

**Comment by mmiddaugh** [comment]
> AC:  If there's an existing test, it is run successfully. If not, stub a ticket to note this for future work.

@laflannery, I'd suggest we need to update existing tests as necessary so that they run successfully as part of this work. 
The need to add tests should be stubbed for the future. 
If I'm missing context or misunderstanding, please let me know.

**Comment by xiongjaneg** [comment]
@mmiddaugh Yes, that AC was standard language we had discussed so I can update it.

**Comment by eselkin** [comment]
I estimate a 2

**Comment by randimays** [comment]
Adding a note here: this page does not meet the criteria for a Back to Top component per [the DS guidelines](https://design.va.gov/components/back-to-top#when-to-use-back-to-top) as it is dynamic content.

<img width="606" alt="Screenshot 2024-05-02 at 2 34 46 PM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/19175324/b5eab2d6-5a5e-4936-8686-98e8bb0c402f">

**Comment by randimays** [comment]
Chatted with @laflannery and Matthew Dingee about the Back to Top components and its usage requirements. [DST conversation here](https://dsva.slack.com/archives/C01DBGX4P45/p1714680374615469). I proposed that DST add some functionality into the Back to Top component ([bug ticket here](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2796)) where it would only appear if the page height meets a minimum height. When they make this change, we can add the BTT component to all VA.gov pages (using the templates in content-build) and it would only appear when necessary.

Circling back around: @laflannery, do we want to add Back to Top to these pages in the meantime (considering DST may not get to prioritize this work soon)?

**Comment by laflannery** [comment]
I think maybe we don't add them to these News pages, because these can be very short, [example](https://www.va.gov/minneapolis-health-care/news-releases/mankato-va-community-clinic-ribbon-cutting-ceremony/). I think this would be more important for the main VAMC pages that this is currently missing on

**Comment by randimays** [comment]
I updated the ACs to reflect this. Thanks @laflannery!

**Comment by laflannery** [comment]
For the video links: Use the standard va-link component

For the Image download links:
- use the download va-link component
- use the `filetype` prop with the extension of the image that has been added
- For the content of the link can we add "Download {the name of the file}"
   - I realize this is going to be at times duplicative when the name of the file also has the file extension in it but I think it would be better to do the component portion of this properly and then we can handle the content entry portion
